message_sender/tasks.py

```diff
@@ -77,6 +79,44 @@ def run(self, metric_name, metric_value, session=None, **kwargs):
 fire_metric = FireMetric()
 
 
+class Concurrency_Limiter(object):
+    def __init__(self):
+        self.redis_server = redis.StrictRedis(host=settings.REDIS_HOST,
+                                              port=settings.REDIS_PORT,
+                                              db=settings.REDIS_DB)
```

Comment:
I'm wondering if instead of using redis directly here, we should rather use django's cache framework? That allows us to easily swap the caching backend between memory, redis, memcached, etc, and takes care of the nitty gritty things.